functions/customgear/fn_onCustomGearTabButton.sqf

```diff
@@ -15,12 +15,12 @@ _ctrlTabSelected ctrlCommit 0;
 
 // get available options for selected category
 private _hashKey = _button getVariable [QGVAR(hashKey), ""];
-private _loadoutOptionsHash = _display getVariable [QGVAR(loadoutOptionsHash), []];
-private _availableOptions = if !([_loadoutOptionsHash] call CBA_fnc_isHash) then {
+private _loadoutOptionsHash = _display getVariable [QGVAR(loadoutOptionsHash), false];
+private _availableOptions = if (_loadoutOptionsHash isEqualTo false) then {
     ERROR("_loadoutOptionsHash got lost along the way and is no longer a hash.");
     []
 } else {
-    [_loadoutOptionsHash, _hashKey] call CBA_fnc_hashGet
+    _loadoutOptionsHash getOrDefault [_hashKey, false];
```

Comment:
Wouldn't false create an issue here, since the default value used to be []?

Reply:
hmm two lines down the comment says the default value should actually be `false`.

Reply:
this is the place its being defined: 

https://github.com/gruppe-adler/grad-loadout/blob/master/functions/customgear/fn_getCustomGearOptions.sqf#L5

and its set here (first marked line: call to get the hash from getCustomGearOptions , last marked line where its set into the display:

https://github.com/gruppe-adler/grad-loadout/blob/6a8d48ee8c1b5319faf5afba7947d82cc9579856/functions/customgear/fn_openCustomGearDialog.sqf#L33-L38